code/game/objects/structures/crimesign.dm

```diff
@@ -0,0 +1,72 @@
+/obj/effect/crimesign
+	name = "security holosign"
+	icon = 'icons/effects/crimesign.dmi'
+	desc = "A laser-holo projected floor marking. This one indicates an active security situation. You probably shouldn't interfere..."
+	icon_state = "crimesign"
+	layer = LOW_OBJ_LAYER
+	plane = FLOOR_PLANE
+	light_color = "#ff2466"
+	light_range = 1
+	light_power = 0.25
+
+/obj/effect/crimesign/Initialize(mapload, turf/source, size)
+	. = ..()
+	alpha = 100
+	add_filter("bloom" , 1 , list(type="bloom", size=0.5, offset = 0.1, alpha = 200))
+	align(source, size)
+	update_appearance(UPDATE_ICON)
```

Comment:
Initialization should automatically call update appearance, but you need to call align() before the ..() call I believe